core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/workflow/impl/LoadSequencerWorkflow.java

```diff
@@ -0,0 +1,59 @@
+package uk.ac.bbsrc.tgac.miso.core.data.workflow.impl;
+
+import static uk.ac.bbsrc.tgac.miso.core.data.workflow.Workflow.WorkflowName.LOAD_SEQUENCER;
+
+import java.io.IOException;
+import java.util.List;
+
+import uk.ac.bbsrc.tgac.miso.core.data.workflow.AbstractWorkflow;
+import uk.ac.bbsrc.tgac.miso.core.data.workflow.ProgressStep;
+import uk.ac.bbsrc.tgac.miso.core.data.workflow.WorkflowExecutor;
+import uk.ac.bbsrc.tgac.miso.core.data.workflow.WorkflowStep;
+import uk.ac.bbsrc.tgac.miso.core.data.workflow.WorkflowStepPrompt;
+
+public class LoadSequencerWorkflow extends AbstractWorkflow {
+  @Override
+  protected List<WorkflowStep> getCompletedSteps() {
+    return null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1168)